packages/kotti-ui/source/kotti-button/types.ts

```diff
@@ -0,0 +1,28 @@
+import { Yoco } from '@3yourmind/yoco'
+
+export namespace KottiButton {
+	export enum Type {
+		PRIMARY = 'primary',
+		SECONDARY = 'secondary',
+		TEXT = 'text',
+		DANGER = 'danger',
+	}
+
+	export enum Size {
+		SMALL = 'small',
```

Comment:
I think this is a breaking change

Reply:
Can you explain what about this is a breaking change? I suppose you think that originally you would pass `"sm"` instead of `"small"` , mainly because @carsoli stated the same in a previous comment. But neither the [documentation](https://3yourmind.github.io/kotti/usage/components/button) nor the [previous code](https://github.com/3YOURMIND/kotti/pull/392/files#diff-fa0908517b6e4a5446821f5f904942271f3a861afdece9b31bf7f0f4d25c92d4L40) would suggest this is the case. Am I missing something here?

Reply:
Wow, I assumed that surely nobody would’ve bothered to explicitly transform `small` into `sm` just to "save" 3 characters in the CSS. In that case, this is fine and preferable.